Update Packages to Latest Development Branches and Apply Compatibility Bug Fixes

nullbyte91 opened the pull request:
This pull request addresses several bug fixes and updates the react_inria download token to ensure compatibility, and up-to-date functionality across different packages and dependencies. The changes made include:

1. Updated the sot-universal-robot branch from 'master' to 'devel'. 
2. Set hpp-pinocchio_commit_id to a839f4e638116867cbcd1072258dd1b2d4128677 to fix issues arising from recent std::shared_ptr changes.
3. Corrected a typo in the agimus-demo_extra_flags variable from a previous commit.
4. Changed the vision_visp branch from 'master' to 'noetic-devel', as the 'master' branch has been removed.
5. Set Universal_Robots_ROS_Driver_commit_id to f56d49ce89d2180e26cfb07b384199dda1fd3912 to maintain compatibility with the removed controller_stopper package.
6. Updated the react_inria download token to ensure the latest version is used.

By implementing these bug fixes and token updates, we aim to enhance the compatibility and stability of the software, while ensuring that users have access to the most recent features and improvements.

nullbyte91 committed:
Fix multiple bugs in various dependencies and update react_inria download token

1. Update sot-universal-robot branch from 'master' to 'devel'
2. Set hpp-pinocchio_commit_id to a839f4e638116867cbcd1072258dd1b2d4128677 for fixing the recent std::shared_ptr changes with other packages.
3. Correct typo in agimus-demo_extra_flags variable in a previous commit.
4. Change vision_visp branch from 'master' to 'noetic-devel', as the 'master' branch has been removed
5. Set Universal_Robots_ROS_Driver_commit_id to f56d49ce89d2180e26cfb07b384199dda1fd3912 to maintain compatibility with the removed controller_stopper package.
6. Update react_inria download token to ensure the latest version is used.

These bug fixes and token update ensure compatibility, stability, and up-to-date functionality across different packages and dependencies.

nullbyte91 committed:
Add  flag to  variable, enabling UR10 pointing installation

florent-lamiraux commented:
Hello nullbyte91

Thank you for this update. However, active token should not be put on github since they give access to private code to anybody. I therefore revoked the token that you published in the Makefile line 1198.

See my other comments in the code.

florent-lamiraux commented on docker/Makefile:
```diff
@@ -672,6 +674,24 @@ update:
 		cat .git/refs/heads/${$(@:.log=)_branch}; \
 	fi
 
+hpp-pinocchio.checkout:
```
Why do you need to redefine those rules. There are default rules for `checkout`, `install`, `update`,...
They only need to be overriden if the default rule does not work. I do not think this is the case here.

nullbyte91 committed:
1. Change Pinocchio branch to v2.6.8 from devel and add USE_SHARED_PTR_FROM_HPP_FCL flag.
2. Switch hpp-fcl branch to devel from react.
3. Update Universal Robots ROS Driver to version 2.1.0 from devel branch.

nullbyte91 committed:
Automate React project installation with a comprehensive shell script

florent-lamiraux commented:
Thank you.
